test/save-speech.spec.js

```diff
@@ -0,0 +1,71 @@
+'use strict';
+const expect = require('chai').expect;
+const app = require('./test-apps/basic-app');
+const fs = require('fs');
+const rimraf = require('rimraf');
+
+describe('saveSpeechAssetsToDirectory', () => {
+
+    before((done) => {
+        fs.stat('speechAssets', (err, stat) => {
+            if (stat) {
+                rimraf.sync('speechAssets');
+            }
+
+            app.saveSpeechAssets();
+            done();
+        });
+    });
+
+    it('should save intentSchema to JSON file', () => {
+        const intentSchema = fs.readFileSync('speechAssets/intentSchema.json', 'utf8');
+        expect(JSON.parse(intentSchema)).to.deep.equal({
```

Comment:
extract these tested values into separate file and require it directly (same goes fore utterances)

do the same for generate.spec.js since the values are identical here (same app is tested)

add it to `tests/mock` or something similar